processor/src/services/mock-payment.service.ts

```diff
@@ -286,6 +275,19 @@ export class MockPaymentService extends AbstractPaymentService {
       paymentId: newlyCreatedPayment.id,
     });
 
+    const transactionState: TransactionState = isBelowSuccessStateThreshold
+      ? TRANSACTION_STATE_SUCCESS
+      : TRANSACTION_STATE_FAILURE;
+
+    await this.ctPaymentService.updatePayment({
```

Comment:
as an improvement, we could generate a random PSP reference as we do in the `createPayment` so we are consistent with that logic and with the expected behaviour from a real PSP

Reply:
Pushed in https://github.com/commercetools/connect-payment-integration-template/pull/422/commits/c49ebdac10e11f249df41a5100b483570e197100